unitTests/ui/DynamicFacet/DynamicFacetTestUtils.ts

```diff
@@ -8,7 +8,7 @@ import { IFacetResponse } from '../../../src/rest/Facet/FacetResponse';
 export class DynamicFacetTestUtils {
   static allOptions(options?: IDynamicFacetOptions) {
     return {
-      field: '@dummy',
+      field: `@dummy${Math.round(Math.random() * 100000)}`,
```

Comment:
If this was to ensure you had different values for the field, I would prefer passing the `options` property to specify the field rather than having a chance no matter how small of the same test passing and failing on successive runs.